### PR TITLE
INFRA: Initialize Api Project

### LINKED ISSUE #8: INFRA: Initialize Api Project

### PR DESCRIPTION
**Pull request by Shrabanac**
Closes #8

### CODE REVIEW

File: JustLearn.Services.Api/JustLearn.Services.Api.csproj [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```
Comment (ShriHumrudha) [inline JustLearn.Services.Api/JustLearn.Services.Api.csproj]: 6.0

File: JustLearn.Services.Api/Startup.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
+using Microsoft.AspNetCore.Builder;
```
Comment (ShriHumrudha) [inline JustLearn.Services.Api/Startup.cs:5]: Copyrights header

File: JustLearn.Services.Api/Startup.cs [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.OpenApi.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
```
Comment (ShriHumrudha) [inline JustLearn.Services.Api/Startup.cs]: Run Code Cleanup

File: JustLearn.Services.Api/Startup.cs [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+namespace JustLearn.Services.Api
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration) =>
+            Configuration = configuration;
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
```
Comment (ShriHumrudha) [inline JustLearn.Services.Api/Startup.cs]: remove commented code

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Review by NaynaMallick — APPROVED** [review]

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**